api/src/main/java/org/openmrs/module/appointments/service/SpecialityService.java

```diff
@@ -15,4 +15,8 @@
     @Transactional
     @Authorized({"View Appointments"})
     List<Speciality> getAllSpecialities();
+    
+    @Transactional
+    @Authorized({"Manage Appointment Services"})
```

Comment:
I believe we should create a specific permission for this "Manage Appointment Specialities"

Reply:
Done